### PR TITLE
Add working classification/autoencoder

### PR DESCRIPTION
**Pull request by DavinDillon**
-substituting the nanmean inside the model using keras -testing against the np.nanmean

### CODE REVIEW

**Comment by DavinDillon** [comment]
@minztmyo

**Review by minztmyo — CHANGES_REQUESTED** [review]
1. Only call imports that are actually used and sort them alphabetically (you can use isort for help).
2. Remove all unnecessary empty lines and empty blocks.
3. Are all the "prints" necessary and informative? eg. type(X) is unnecessary
4. Make use of helpful jupyter cell types, like markdown. Describe, with titles (#), subtitles (##), and subsubtitles (###), what you are doing: Load data, Create custom loss function...
5. Too much commented code makes the rest of the code hard to read. You can delete some of those functions that does the same thing. Maybe can also store it in another file and put it in .gitignore if you want to reuse.
6. Comments should have the same indentation as the rest of the surrounding code.
7. Double quotation marks > single quotation marks.
8. Indentation in Python should always be 4 not 2.
9. There should be no need for backslashes in Python.
10. Try to follow PEP8 in general
11. Try to follow Git commit message guidelines: https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53.
etc
12. What is the function in Line 75 for?
13. Line 102: You can delete early stop if you don't use it.
14. Ln 106, huge loss drop. Maybe plot from epoch 2 for better visualization
15. See if those huge Epoch chunks are scrollable. Let's see if it reflects on github too
@DavinDillon
